_button.scss

```diff
@@ -0,0 +1,71 @@
+// button
+// ===
+
+// Configurable Variables
+// ---
+
+//Button Styles
+//---
+
+$button__padding: $vertical-input-padding $horizontal-input-padding !default;
+
+$button__border: if(variable-exists(border), $border, 1px solid $neutral-50) !default;
+$button__border-radius: if(variable-exists(border-radius), $border-radius, 0) !default;
+
+
+$button__font-color: inherit !default;
+$button__font-family: inherit !default;
+$button__font-size: inherit !default;
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+
+
+// Button root
+// ---
+
+.c-button {
+    min-height: $tap-size;
+    display:inline-block;
+    margin: 0;
+    cursor:pointer;
+    background: none;
+
+    text-align:center;
+    border:none;
+    line-height: normal;
+    white-space: normal;
+
+    span {
```

Comment:
An element selector for the visual part of the button might be OK, since the dust template can mostly enforce the DOM structure. However, if we do, it should definitely be `.c-button > span`, otherwise any other descendants (like `<span class="c-icon">` will end up getting messed up. If we want to be strict, this element should get a class and be styled with a class selector. Just not sure what to call it – probably `c-button__inner` even though names like that make me cringe.